lib/src/__tests__/DatePickerTestingLib.test.tsx

```diff
@@ -1,81 +1,116 @@
+import 'moment/locale/fr';
+import 'moment/locale/ru';
+import 'moment/locale/ar-sa';
 import * as React from 'react';
+import moment from 'moment';
+import jMoment from 'moment-jalaali';
+import ruLocale from 'date-fns/locale/ru';
+import frLocale from 'date-fns/locale/fr';
+import MomentAdapter from '@date-io/moment';
+//import JalaliAdapter from '@date-io/jalaali';
+import DateFnsUtils from '@date-io/date-fns';
+import enLocale from 'date-fns/locale/en-US';
 import TextField from '@material-ui/core/TextField';
-import { getByMuiTest } from './test-utils';
-import { screen, waitFor } from '@testing-library/react';
-import { utilsToUse, FakeTransitionComponent } from './test-utils';
-import { createClientRender, fireEvent } from './createClientRender';
-import { DatePicker, MobileDatePicker, DesktopDatePicker } from '@material-ui/pickers';
+import LocalizationProvider, {
+  LocalizationProviderProps,
+} from '@material-ui/pickers/src/LocalizationProvider';
```

Comment:
From what I understand, we don't support nested imports yet as we do with the other packages.

```suggestion
import { LocalizationProvider, LocalizationProviderProps } from '@material-ui/pickers';
```